### PR TITLE
VertexShaderGen: Remove the need for an extra UID.

### PR DESCRIPTION
**Pull request by Armada651**
We don't actually need add a new UID to disable the vertex depth range calculations, we can achieve the same thing by changing the variables.

### CODE REVIEW

**Comment by BhaaLseN** [comment]
Shouldn't this change some cache version somewhere?

**Comment by Armada651** [comment]
I didn't know we had cache versions? I didn't change the version in my last PR that introduced this UID either.

**Comment by BhaaLseN** [comment]
I thought we had one for the shader cache too, but I guess I was wrong. Could only find references for Save States, Gamelist etc.

File: Source/Core/VideoCommon/VertexShaderGen.cpp [hunk 1]
```diff
@@ -438,23 +434,15 @@ ShaderCode GenerateVertexShaderCode(APIType api_type, const vertex_shader_uid_da
...
-              "o.pos.z * " I_PIXELCENTERCORRECTION ".z;\n");
-  }
-  else
-  {
-    // Here we rely on the host driver to process the depth range, however we still need to invert
-    // the console -1..0 range to the 0..1 range used in the depth buffer.
-    out.Write("o.pos.z = -o.pos.z;\n");
-  }
+  //
+  // Adjust z for the depth range. We're using an equation which incorperates a depth inversion,
```
Comment (Sintendo) [inline Source/Core/VideoCommon/VertexShaderGen.cpp]: Might as well fix the spelling of 'incorporates' while your in the neighbourhood.

**Pull request merged**